.travis.yml

```diff
@@ -1,14 +1,13 @@
 language: node_js
 node_js:
-  - "0.10"
-  - "0.12"
   - "4"
   - "5"
+  - "6"
```

Comment:
Looks like leveldown, both the `0.10.6` linked by `level-test@~1.7.0` and `1.4.6` from `leveldown@~1.4.4`,  have problems building on v7. Removing for now, we can try to address in #38 ?